src/Lykke.Service.ExchangeDataStore/Lykke.Service.ExchangeDataStore.Core/Settings/ServiceSettings/RabbitMqExchangeConfiguration.cs

```diff
@@ -0,0 +1,15 @@
+﻿// ReSharper disable UnusedAutoPropertyAccessor.Global
+// ReSharper disable ClassNeverInstantiated.Global
+namespace Lykke.Service.ExchangeDataStore.Core.Settings.ServiceSettings
+{
```

Comment:
Probably can be shared with the ExchangeConnector project via a new shared lib?

Reply:
We can do it, but don't you think this couples the services - the new one and ExchangeConnector? Unless its a separately managed nuget package, but that would probably be too much for a single config class? Also, as far as I know, the idea is the new DataStore service to go into its own solution sometime in the future - than will either have to copy the config class (as it is in the moment) or start supporting a new nuget lib?